src/DotNetWorker.Grpc/GrpcWorker.cs

```diff
@@ -258,7 +261,7 @@ internal static WorkerInitResponse WorkerInitRequestHandler(WorkerInitRequest re
             response.Capabilities.Add("TypedDataCollection", bool.TrueString);
             response.Capabilities.Add("WorkerStatus", bool.TrueString);
             response.Capabilities.Add("HandlesWorkerTerminateMessage", bool.TrueString);
-
+            response.Capabilities.Add("EnableUserCodeException", enableUserCodeExceptionOption.ToString());
```

Comment:
Since you made _workerOptions available globally in this file, you should be able to just use it here directly for `_workerOptions.Value.EnableUserCodeException` instead of changing the method to take in a new parameter

e.g. something like

```csharp
var enableUserCodeException = _workerOptions.Value.EnableUserCodeException ?? false;
response.Capabilities.Add("EnableUserCodeException", enableUserCodeException.ToString());
```

Reply:
I think because it's a static method the instance variables can't be accessed directly.

Reply:
Want to understand how cx can set this workerOptions? @madelinegordon told me it's from bootstrapping code, but I am not quite aware of that. Wondering do we have document explaining this. Thanks.